hazelcast/src/test/java/com/hazelcast/ringbuffer/RingbufferSplitBrainTest.java

```diff
@@ -72,25 +81,47 @@
                 {BINARY, PassThroughMergePolicy.class},
                 {BINARY, PutIfAbsentMergePolicy.class},
 
```

Comment:
Could you also add a test for the new `RemoveValuesMergePolicy` (its part of `SplitBrainTestSupport`)? This should test the new code path, when the merge policy returns `null`. I found a heck of bugs in my implementations with it.

Reply:
I added that policy and I also added a ringbuffer store as you did for the queue which actually uncovered a nasty bug so kudos for including the queue store in the tests :)